Makefile

```diff
@@ -8,7 +8,7 @@ build:
 	cd apps/petstore-builder; go get; go install
 	cd plugins/gnostic-summary; go get; go install
 	cd plugins/gnostic-analyze; go get; go install
-	cd plugins/gnostic-go-generator; go get; go install
+	go get github.com/googleapis/gnostic-go-generator
```

Comment:
Is this needed now? Maybe not since we decided not to test the generator in the main repo.

Reply:
I'll remove it